examples/multimodel/modular_speechllm/materialize_text_data.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Do you need this file? Looks like dev code.
If you do, consider moving it to the scripts/ folder

Reply:
done